### PR TITLE
Add abort by timeout

### PR DESCRIPTION
**Pull request by MegaDiablo**

### CODE REVIEW

File: src/qwest.js [hunk 1]
```diff
@@ -142,10 +146,16 @@ module.exports = function() {
 				if(xhr2 || xdr) {
 					xhr.onload = handleResponse;
 					xhr.onerror = handleError;
+					xhr.ontimeout = handleError;
 				}
 				else {
+					var timeout = setTimeout(function() {
+						xmlhttp.abort();
```
Comment (pyrsmk) [inline src/qwest.js]: Is it really `xmlhttp` or `xhr`?

Comment (MegaDiablo) [inline src/qwest.js]: oops, sorry my mistake

**Comment by pyrsmk** [comment]
Did you test your addition? Or have you got an URL that you know can be really long to respond?

**Comment by MegaDiablo** [comment]
I tested xhr2 type of connection. I have long response from my local router http://192.168.1.1 or other pc in local network.

If you give me plan testing I can make full testing :)

**Comment by pyrsmk** [comment]
To make your tests, you can simply : 

- install node modules into qwest with `npm i`
- compile qwest with `gulp build`
- pick the `qwest.min.js` file
- put it in a folder where you've created a sample html file to test things

**Comment by MegaDiablo** [comment]
Tested. Works fine

**Comment by pyrsmk** [comment]
Neat! I'll put all current issues and this PR in the next release (coming really soon)

**Pull request merged**